specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```diff
@@ -0,0 +1,591 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-11-preview",
+    "title": "Microsoft.Quota",
+    "description": "Microsoft Azure Quota Resource Provider."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Quota/providers/{providerId}/locations/{location}/usages": {
```

Comment:
Can you add examples for all these APIs?

Reply:
I will add examples.